packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -286,6 +286,7 @@ export async function getServerSideProps({
               id: userId,
             },
           )
+          .then((data) => data?.user)
```

Comment:
below you have `referringUser: referringUser?.user || null` so it needs to be adjusted

Reply:
Right. Let me update it quickly.

Reply:
To ensure we won't miss anything, I applied some strict types.